import.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+  "github.com/Shopify/sarama"
+  "github.com/golang/protobuf/proto"
+  "github.com/bmeg/arachne/aql"
+  "github.com/ohsu-comp-bio/mortar/events"
+  "github.com/ohsu-comp-bio/tes"
```

Comment:
you don't need to import this if your including the proto file and generated code in this repo